xcode/Subconscious/Shared/Services/DataService.swift

```diff
@@ -663,13 +663,7 @@ actor DataService {
         let identity = try await noosphere.identity()
         var feed: [EntryStub] = []
         for entry in try self.database.listFeed(owner: identity) {
```

Comment:
We can go a step further here and actually do a query with a `JOIN` to ensure each `EntryStub` comes with the user's profile data already.

A `JOIN` has the _disadvantage_ of loading a bunch of duplicate data but it would do it all in one sweep. Currently, we maintain an in-memory cache of the `UserProfile`s for a given `(Did, Petname)` combination so while we do have to call `identifyUser` for every item, the calls become cheap quickly and never hit the underlying sphere.